plugins/output/flight/flight.go

```diff
@@ -0,0 +1,149 @@
+package flight
+
+import (
+	"context"
+	_ "embed"
+	"net"
+
+	"github.com/apache/arrow/go/v9/arrow"
+	"github.com/apache/arrow/go/v9/arrow/array"
+	"github.com/apache/arrow/go/v9/arrow/flight"
+	"github.com/apache/arrow/go/v9/arrow/ipc"
+	"github.com/apache/arrow/go/v9/arrow/memory"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+//go:embed sample.conf
```

Comment:
Missing a space at the beginning of this comment.

Reply:
As explained [here](https://pkg.go.dev/cmd/compile#hdr-Compiler_Directives), this is not a literal comment but actually, a compiler directive used to embed that specific file. Their missing space at the beginning of the comment is recognized explicitly as directives.